### PR TITLE
Update SIG Recommener contacts

### PR DESCRIPTION
**Pull request by smilingday**
Update contacts and adjust out-of-date wordings

### CODE REVIEW

File: sigs/recommenders/CHARTER.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Notice that TensorFlow has open-sourced [TensorFlow Recommenders](https://blog.t
 [github.com/tensorflow/recommenders](http://github.com/tensorflow/recommenders)
 
 Further, we maintain a Tensorflow repo dedicated for community contributions and maintained by SIG as well, under: 
-**github.com/tensorflow/recommenders-addons** (to be created).
+**github.com/tensorflow/recommenders-addons**.
```
Comment (zenogantner) [inline sigs/recommenders/CHARTER.md:25]: Would it make sense to turn this into a proper URL, now that the repository exists?

I.e., https://github.com/tensorflow/recommenders-addons

File: sigs/recommenders/CHARTER.md [hunk 2]
```diff
@@ -33,24 +33,26 @@ Everybody with an interest in using TensorFlow to build recommendation systems a
 
 - SIG Recommenders [mailing list](https://groups.google.com/u/1/a/tensorflow.org/g/recommenders)
 - New TensorFlow github maintained by SIG Recommenders
-(github.com/tensorflow/recommenders-addons, to be created)
+(github.com/tensorflow/recommenders-addons)
```
Comment (zenogantner) [inline sigs/recommenders/CHARTER.md:36]: Suggestion for replacing the 2 lines above with:
´´´
 - GitHub repository maintained by SIG Recommenders: https://github.com/tensorflow/recommenders-addons
´´´

**Comment by smilingday** [comment]
Looks good, thanks for doing this.

@theadactyl to approve this.

**Review by theadactyl — APPROVED** [review]
LGTM

**Pull request merged**